### PR TITLE
Add Participatory processes (1/2)

### PR DESCRIPTION
**Pull request by beagleknight**
# What and why

We are going to add a feature to organize `proposals`, `meetings`, `debates`, `categories` and `subcategories` in a group called `ParticipatoryProcess`.

This PR includes the data layer and the administration to manage participatory processes.
# GIF Tax

### CODE REVIEW

**Comment by josepjaume** [comment]
We should write some tests to verify this is working. Acceptance and maybe unit for `ParticipatoryProcess`.

File: app/controllers/admin/participatory_processes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  def destroy
+    @participatory_process.destroy
+    redirect_to admin_participatory_processes_url, notice: t('flash.actions.destroy.notice', resource_name: "ParticipatoryProcess")
+  end
+
+  private
+
+  def strong_params
+    send("participatory_process_params")
```
Comment (josepjaume) [inline app/controllers/admin/participatory_processes_controller.rb:42]: W0t?

**Comment by josepjaume** [comment]
Looks fine!

**Comment by josepjaume** [comment]
I believe this can be closed in favor of https://github.com/AjuntamentdeBarcelona/decidim.barcelona/pull/434

File: app/controllers/admin/participatory_processes_controller.rb (hunk 1 above)
Comment (beagleknight) [inline app/controllers/admin/participatory_processes_controller.rb:42]: wut? XD

**Comment by beagleknight** [comment]
I'm agree @josepjaume. Closing this :smile:

**Pull request closed without merging**